fabric-item-api-v1/src/main/java/net/fabricmc/fabric/api/item/v1/FabricItemSettings.java

```diff
@@ -54,14 +54,14 @@ public FabricItemSettings customDamage(CustomDamageHandler handler) {
 	}
 
 	/**
-	 * Sets this item as a fuel, with a provided {@code cookTime}.
+	 * Sets this item as a fuel, with a provided {@code burnTime}.
 	 *
-	 * @param cookTime the number of ticks before this item will be consumed in a furnace
+	 * @param burnTime the number of ticks before this item will be consumed in a furnace fuel slot
```

Comment:
A bit misleading - the item is consumed directly if it's the only one in the slot. Imo you can just omit this `@param` as it's obvious.